nemo/collections/diffusion/models/dit/dit_layer_spec.py

```diff
@@ -695,7 +695,7 @@ def __init__(
     ):
         super().__init__(config=config, submodules=submodules, layer_number=layer_number)
 
-        if config.enable_cuda_graph:
+        if config.enable_cuda_graph and config.cuda_graph_scope != "full_iteration":
```

Comment:
Please a comment of `Enable per-Transformer layer cuda graph` above

Reply:
Done